lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -122,6 +122,14 @@ export class PublicClientApplication {
         }
```

Comment:
Can you add a warning in `handlRedirectCallback` indicating it is deprecated, will be removed in the stable version, and that the new promise should be used instead?

Reply:
done.